Overview.src.html

```diff
@@ -3082,6 +3082,31 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
     params = {lat:35.696233, long:139.570431}
 Object.keys(params).forEach(key => url.searchParams.append(key, params[key]))
 fetch(url).then(/* &hellip; */)</pre>
+
+ <p>If you want to receive the body data progressively:
+
+ <pre>function consume(reader) {
+  var total = 0
+  return new Promise((resolve, reject) => {
+    function pump() {
+      reader.read().then(({done, value}) => {
+        if (done) {
+          resolve()
+          return
+        }
+        total += value.byteLength
+        log("received " + value.byteLength + " bytes (" + total + " bytes in total).")
```

Comment:
Optional ES6 goodness: ``log(`received ${value.byteLength} bytes (${total} bytes in total)`)``.